.github/workflows/e2e-test.yaml

```diff
@@ -17,18 +17,21 @@ on:
           - k3d
           - kind
         default: k3d
-  workflow_call:
-    inputs:
-      kernel-image:
-        type: string
-        description: 'The kernel image to use for the VMs'
-        required: false
 
 env:
   PRESERVE_RUNNER_PODS: "true"
 
 jobs:
+  vm-kernel:
+    uses: ./.github/workflows/vm-kernel.yaml
```

Comment:
Out of curiosity: How does `uses` work here? does it inline the entire contents of `vm-kernel.yaml`, or does it only use the `vm-kernel` job from `vm-kernel.yaml` ?
(and then, later on, when we say `vm-kernel.outputs`, does that refer to the job, or the entire workflow?)

Reply:
It calls all jobs from `vm-kernel.yaml`, and `vm-kernel.outputs` refers to workflow outputs. So, there's no external access to jobs in `vm-kernel.yaml` workflow.

Reply:
Makes sense - thanks!